### PR TITLE
Radh 2

### PR DESCRIPTION
**Pull request by someone20dz**
here is my work 

- [x] Date of Conference (If Time Permits: Timeline Effect)
- [x] Conference Timings
- [x] Layout and Rules
- [x] Goal of Conference
- [x] Intended Audience
- [x] Scope of Speakers and Events

### CODE REVIEW

**Comment by someone20dz** [comment]
the rest of the work isn't done still need information

**Review by JAnanth — APPROVED** [review]

**Review by TheChanRProject — CHANGES_REQUESTED** [review]
Fix typography for logo and ChanR title in About page. 

Change background image of timeline to a colored gradient.

File: chanr_hs_immersive.html [hunk 1]
```diff
@@ -259,39 +262,98 @@ <h5> The Data & Software Immersion Summit is the first virtual conference specif
...
-             <div class="header-2">
-               <div class="page-header" style="background-image: url('assets/img//chanr_immersive/cool_tech_img.jpg');">
-                 <div class="filter"></div>
-                 <div class="content-center">
-                   <div class="container">
-                     <!-- Conference Title !-->
-                     <!-- Logo with Mission and Links !-->
-                            <div class="row">
-                              <div class="col-md-6 text-left">
-                                <h1 class="library-3am-title ct-azure">Conference</h1>
```
Comment (TheChanRProject) [inline chanr_hs_immersive.html:283]: Add comments bro

File: chanr_hs_immersive.html [hunk 2]
```diff
@@ -259,39 +262,98 @@ <h5> The Data & Software Immersion Summit is the first virtual conference specif
...
+                                                              <p>Registration Deadline(Early Bird Pricing)</p>
+                                                            </div>
+                                                          </div>
+                                                          <div class="container m_right">
+                                                            <div class="my_content">
+                                                              <h3>June 12 2021</h3>
+                                                              <p>Event Date</p>
+                                                            </div>
+                                                          </div>
+                                                        </div>
```
Comment (TheChanRProject) [inline chanr_hs_immersive.html:313]: Structure looks ok, but I'm wondering if the tag structuring is easy to follow or not. Might be hard for newcomer to follow along.

File: chanr_hs_immersive.html [hunk 3]
```diff
@@ -259,39 +262,98 @@ <h5> The Data & Software Immersion Summit is the first virtual conference specif
...
+                              </a>
                             </div>
                           </div>
                         </div>
                       </div>
                     </div>
-                    </div>
+                  </div>
+                </div>
+              </div>
```
Comment (TheChanRProject) [inline chanr_hs_immersive.html:345]: This is very nice. Lovely use of gradient for the background.

File: chanr_hs_immersive.html [hunk 4]
```diff
@@ -2603,11 +2665,11 @@ <h5 class="info-title">Find us at the office</h5>
...
+  <script src="./assets/js/plugins/moment.min.js"></script><!-- Plugin for Tags, full documentation here: https://github.com/bootstrap-tagsinput/bootstrap-tagsinputs  -->
   <script src="./assets/js/plugins/bootstrap-tagsinput.js"></script>
-  <!--	Plugin for Select, full documentation here: http://silviomoreto.github.io/bootstrap-select -->
+  <!--  Plugin for Select, full documentation here: http://silviomoreto.github.io/bootstrap-select -->
   <script src="./assets/js/plugins/bootstrap-selectpicker.js" type="text/javascript"></script>
-  <!--	Plugin for Datetimepicker, full documentation here: https://eonasdan.github.io/bootstrap-datetimepicker/ -->
+  <!--  Plugin for Datetimepicker, full documentation here: https://eonasdan.github.io/bootstrap-datetimepicker/ -->
   <script src="./assets/js/plugins/bootstrap-datetimepicker.js" type="text/javascript"></script>
   <!--  Plugin for presentation page - isometric cards  -->
   <script src="./assets/js/plugins/presentation-page/main.js"></script>
```
Comment (TheChanRProject) [inline chanr_hs_immersive.html:2675]: Great! I like it

**Review by TheChanRProject — APPROVED** [review]
Looks great, @someone20dz .

Approved. 👍

File: chanr_hs_immersive.html (hunk 2 above)
Comment (someone20dz) [reply chanr_hs_immersive.html:313]: I can help explaining it was tricky to put it in this way in order to work

Comment (TheChanRProject) [reply chanr_hs_immersive.html:313]: Thanks @someone20dz . It is looking nice overall though. 

I'll work on Speakers Page tomorrow.

**Pull request merged**